lib/collection/src/operations/config_diff.rs

```diff
@@ -108,18 +108,25 @@ pub struct OptimizersConfigDiff {
     /// Note: 1Kb = 1 vector of size 256
     #[serde(alias = "max_segment_size_kb")]
     pub max_segment_size: Option<usize>,
-    /// Maximum size (in KiloBytes) of vectors to store in-memory per segment.
+    /// Maximum size (in kilobytes) of vectors to store in-memory per segment.
     /// Segments larger than this threshold will be stored as read-only memmaped file.
-    /// To enable memmap storage, lower the threshold
+    ///
+    /// To enable memmap storage, lower the threshold.
+    /// To disable memmap storage entirely, set to `0`. It will be treated as the largest value possible.
+    ///
     /// Note: 1Kb = 1 vector of size 256
     #[serde(alias = "memmap_threshold_kb")]
     #[validate(range(min = 1000))]
     pub memmap_threshold: Option<usize>,
-    /// Maximum size (in KiloBytes) of vectors allowed for plain index.
-    /// Default value based on <https://github.com/google-research/google-research/blob/master/scann/docs/algorithms.md>
-    /// Note: 1Kb = 1 vector of size 256
+    /// Maximum size (in kilobytes) of vectors allowed for plain index.
+    /// Default value based on <https://github.com/google-research/google-research/blob/master/scann/docs/algorithms.md>.
+    ///
+    /// Recommendend minimum value is `1000`.
+    ///
+    /// When set to `0`, plain index will be disabled. It will be treated as the largest value possible.
```

Comment:
```suggestion
    /// When set to `0`, vector index will be disabled. It will be treated as the largest value possible.
```

Reply:
> It will be treated as the largest value possible.

didn't get that this means

Reply:
What I want to say is that setting it to `0`, internally means setting it to `u64::MAX`